service/src/test/java/uk/gov/hmcts/reform/fpl/service/CaseValidatorServiceTest.java

```diff
@@ -174,13 +181,30 @@ void shouldNotReturnAnErrorWhenFirstRespondentHasFullNameButNotSecondRespondent(
         CaseData caseData = partiallyCompleteCaseData()
             .grounds(grounds())
             .applicants(applicants(true))
+            .solicitor(solicitor())
             .respondents1(respondents())
             .build();
 
         List<String> errors = caseValidatorService.validateCaseDetails(caseData);
         assertThat(errors).isEmpty();
     }
 
+    @Test
+    void shouldReturnAnErrorWhenSolicitorDoesNotHaveNameAndEmail() {
```

Comment:
Do we need to test this separately? You've asserted this twice above no?

Reply:
Good point will remove